src/components/Quotes.js

```diff
@@ -0,0 +1,66 @@
+import { useState, useEffect } from 'react';
+
+const Quotes = () => {
+  const [quote, setQuote] = useState([{ author: '', quote: '', category: '' }]);
+  const [error, setError] = useState(null);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    const getQuote = async () => {
+      try {
+        const apiKey = 'lqhRH3DsQ3XaCW6r/T99zQ==T8jrMwq87vPiHJPk';
+        const apiUrl = 'https://api.api-ninjas.com/v1/quotes?category=change';
+        const response = await fetch(apiUrl, {
+          headers: {
+            'Content-Type': 'application/json',
+            'X-Api-Key': apiKey,
+          },
+        });
+        const data = await response.json();
+        setQuote(data[0]);
+        setLoading(false);
+      } catch (error) {
+        setError(error.message);
+        setLoading(false);
+      }
+    };
+    getQuote();
+  }, []);
+
+  if (error) {
+    return (
+      <div className="error">
+        Error! Something went wrong:
+        {error}
+      </div>
+    );
+  }
+
+  if (loading) {
+    return <div className="loading">Loading...</div>;
+  }
+
+  return (
+    <div className="quote-block">
+      <div className="quote">
+        {quote.quote}
+        <br />
+        <br />
+        By
+        {' '}
+        <b>{quote.author}</b>
+        <br />
+        <br />
+        You can find more
+        {' '}
+        <b>{quote.category}</b>
+        {' '}
+        quotes in
+        {' '}
+        <a href="https://api.api-ninjas.com/v1/quotes?category=change">here</a>
```

Comment:
- [ ] Great job implementing the quotes component :clap: However when click on the link to see more quotes under a category, the App return a error (See image). Therefore I suggest you simply remove the link since it is not part of the requirement or make sure it works as it intend to.
![Screenshot from 2023-08-18 07-10-34](https://github.com/Arch-Noize/math-magicians/assets/91142432/34052183-6475-4300-9a29-1d01eef923c8)